src/App.js

```diff
@@ -1,24 +1,35 @@
 import React, { Component } from 'react';
 import Area from './movie-area.js';
 import Header from './header.js';
-import movieData from './mock-data.js';
 import Details from './movie-details.js';
+import { getMovie, getVideo } from './api-calls';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
       movies: [],
       currentMovie: null,
+      currentVideo: null,
+      error: '',
     };
   }
 
   componentDidMount() {
-    this.setState({movies: movieData.movies})
+    getMovie()
```

Comment:
The function is called getMovie (singular) but appears to get all movies? I would consider changing that to match.